### PR TITLE
Force query to look in DB

### PR DESCRIPTION
**Pull request by octosteve**
This removes having to do a `reload` after a user unlikes a picture.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.2.0'
+ruby '2.2.3'
```
Comment (joshuabamboo) [inline Gemfile]: Should we be specifying a ruby version in the gemfile cc: @jmburges @StevenNunez

File: Gemfile [hunk 2]
```diff
@@ -1,5 +1,4 @@
 source 'https://rubygems.org'
-ruby '2.2.0'
```
Comment (joshuabamboo) [inline Gemfile:2]: Should we be specifying a ruby version in the gemfile?  cc: @StevenNunez @jmburges

Comment (octosteve) [inline Gemfile:2]: I say no. The students will likely have the latest version meaning we'll constantly be changing the version on labs https://gist.github.com/fnichol/1912050

Comment (joshuabamboo) [inline Gemfile:2]: :+1: merging

**Pull request merged**